lms/migrations/versions/52755322151e_db_migration_to_back_fill_organizations.sql

```diff
@@ -59,5 +59,5 @@ SELECT
     ids_and_guids_right.application_instance_id AS right_id
 FROM ids_and_guids AS ids_and_guids_left
 JOIN ids_and_guids AS ids_and_guids_right
-    ON ids_and_guids_left.application_instance_id = ids_and_guids_right.application_instance_id
+    ON ids_and_guids_left.guid = ids_and_guids_right.guid
```

Comment:
This just resulted in every application instance being on it's own... we're trying to find different application instances with the same guid, not different guids with the same application instance.